pandas/tests/plotting/test_hist_method.py

```diff
@@ -253,6 +253,23 @@ def test_tight_layout(self):
 
         tm.close()
 
+    def test_hist_subplot_xrot(self):
+        df = DataFrame(
```

Comment:
can you add the issue number here as a comment

Reply:
Thanks @jreback. I have added the issue number.